src/sif/autocost.cc

```diff
@@ -262,12 +266,14 @@ class AutoCost : public DynamicCost {
    * @param  node  Node (intersection) where transition occurs.
    * @param  pred  the opposing current edge in the reverse tree.
    * @param  edge  the opposing predecessor in the reverse tree
+   * @param  has_flow_speed Do we have any of the measured speed types set?
    * @return  Returns the cost and time (seconds)
    */
   virtual Cost TransitionCostReverse(const uint32_t idx,
                                      const baldr::NodeInfo* node,
                                      const baldr::DirectedEdge* pred,
-                                     const baldr::DirectedEdge* edge) const override;
+                                     const baldr::DirectedEdge* edge,
+                                     const bool has_flow_speed) const override;
```

Comment:
This is a TODO:  We should try to update TransitionCostReverse with `const EdgeLabel& pred` vs `const baldr::DirectedEdge* pred`; however, we did not have time to make this change.  

@dnesbitt noted that when expanding in reverse the edge added to the edge label is in the direction of expansion.  This is the reverse of the direction of travel. This is why the opposing predecessor directed edge is passed into the TransitionCostReverse method rather than the predecessor EdgeLabel. With some thought and care it may be possible to refactor this but I'm not sure. We need to make sure idx is computed correctly is one complexity